### DISCUSSION TITLE
无法使用chartero（阅读进度无法显示）

### DISCUSSION THREAD

**Discussion by tina140119** [volatile-static/Chartero#82] [Q&A] [answered]
![1704425199853](https://github.com/volatile-static/Chartero/assets/115853966/00681f65-44e7-487f-a65f-62d1185e0562)
最近刚更新zotero beta版本，下载最新的chartero，发现阅读文献后阅读进度无法显示（重启软件也无法解决，重新安装插件也无法解决，安装稍微低版本的插件也无法解决），但其余的插件都能正常运行。
因为装的插件有好几个，请问是否存在插件冲突问题，以及如何解决。蟹蟹作者大大~

**Comment by volatile-static** [comment, marked as answer]
1. 禁用所有其他插件
2. 检查首选项中记录周期是否过长
3. 检查“阅读历史记录”条目下有无数据

**Reply by tina140119** [reply]
已解决，蟹蟹大大。
1. 首先禁用了其他所有的插件
2. 检查首选项中，发现记录周期为10000s，调回正常数值后，发现记录功能恢复正常
3. 最开始阅读历史条目无相关记录，但是把记录周期改回正常值后，已有显示
4. 重新打开所有其他插件，发现chartero和其他插件并无冲突

总结：总算可以愉快使用好用的插件了，蟹蟹大大，祝大大新年快乐，一帆风顺！

**Comment by 25461235461** [comment]
我也遇到了这个插件的困难，早就更新了版本7，使用一段时间，但是我的阅读时长显示全部消失了，阅读记录找不回来，自动查找失败，手动导入说格式不对，请问如何解决

**Reply by volatile-static** [reply]
截图看一下你手动的格式

**Reply by 25461235461** [reply]
![Uploading 图片3.png…]()

**Reply by 25461235461** [reply]
chartero#2PQ74G6H {"pages":{"0":{"p":{"1732258405":1,"1732258406":1,"1732258407":1,"1732258408":1,"1732258410":1,"1732258411":1,"1732258412":1,"1732258413":1,"1732258414":1}}},"numPages":59}

**Reply by 25461235461** [reply]
就是这个，直接复制过去，不懂这个格式

**Reply by 25461235461** [reply]
![图片3](https://github.com/user-attachments/assets/3d71013b-e33b-4259-921b-c9c676ad6d3c)

**Reply by volatile-static** [reply]
你这个是z7版本的格式，导入只能是z6版的
